lib/jsdom/living/nodes/SVGElement.idl

```diff
@@ -0,0 +1,6 @@
+interface SVGElement : Element {
```

Comment:
They can be omitted. Apparently the SVG working group thinks removing them is an acceptable change for SVG2: https://svgwg.org/svg2-draft/changes.html#types